src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -584,14 +584,6 @@ static inline int MPIDI_NM_mpi_put(const void *origin_addr,
         goto fn_exit;
     }
 
-    MPIDI_CH4U_EPOCH_CHECK_SYNC(win, mpi_errno, goto fn_fail);
```

Comment:
Yes, only the `MPI_PROC_NULL` check should be at the CH4 level. The sync functions should remain in the netmod.